rep-2000.rst

```diff
@@ -172,6 +172,128 @@ Middleware Implementation Support:
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| CMake       |     3.10.2     |     3.12.2    |     3.12.1     |       3.5.1       |        3.7.2        |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| EmPY        |     3.3.2      |     3.6.5     |     3.3.2      |       3.3.2       |        3.3.2        |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| Ogre        |     1.10*      |     1.10*     |     1.10*      |       1.10*       |        1.10*        |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| OpenCV      |     3.2.0      |     3.4.1     |     3.4.1*     |       2.4.9       |        3.2*         |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| Poco        |     1.8.0      |     1.9.0     |     1.8.0*     |       1.8.0*      |        1.8.0*       |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| Python      |     3.6.5      |     3.7.0     |     3.7.0      |       3.5.1       |        3.5.3        |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| Qt          |     5.9.5      |     5.11.2    |     5.10.0     |       5.5.1       |        5.7.1        |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+|                              |           Linux only           |                                         |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| PCL         |     1.8.1      |     N/A       |     N/A        |       1.7.2       |        1.8.0        |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+|                                  RMW DDS Milddleware Providers                                          |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| Connext DDS |     5.3.1      |     5.3.1     |     5.3.1      |       5.3.1       |         N/A         |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+|  Fast-RTPS  |     1.7.0      |     1.7.0     |     1.7.0      |       1.7.0       |        1.7.0        |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
+| OpenSplice  | 6.9.181126OSS  | 6.9.181126OSS | 6.9.181126OSS  |   6.9.181126OSS   |    6.9.181126OSS    |
++-------------+----------------+---------------+----------------+-------------------+---------------------+
```

Comment:
GitHub doesn't render this table well. Not sure it that works when merged or if the source needs tweaking.

Reply:
Resolved with e0cbf5d